kiwi/boot/image/dracut.py

```diff
@@ -111,10 +130,10 @@ def write_system_config_file(self, config, config_file=None):
                 )
             )
         if dracut_config:
-            with open(config_file, 'w') as config:
-                config.writelines(dracut_config)
+            with open(config_file, 'w') as config_fd:
+                config_fd.writelines(dracut_config)
```

Comment:
Same here, that change is imho not related to type hints ?

Reply:
It is, `config` is declared as `Dict[str, str]` but here we are assigning a `FileIO` to it, which is a violation of static types. That's why I changed the variable name.

Reply:
Oh I see, yes makes sense.